src/NServiceBus.Core/DetectObsoleteConfigurationSettings.cs

```diff
@@ -43,6 +44,19 @@ static void DetectObsoleteConfiguration(UnicastBusConfig unicastBusConfig)
             }
         }
 
+        static void DetectObsoleteConfiguration(AuditConfig auditConfig)
+        {
+            if (!string.IsNullOrWhiteSpace(auditConfig?.QueueName))
+            {
+                Logger.Warn($"The use of the {nameof(AuditConfig.QueueName)} attribute in the {nameof(AuditConfig)} configuration section is discouraged and will be removed in the next major version. Remove this from the configuration section. Switch to the code API by using `{nameof(EndpointConfiguration)}.AuditProcessedMessagesTo` instead.");
+            }
+
+            if (auditConfig?.OverrideTimeToBeReceived != null)
+            {
+                Logger.Warn($"The use of the {nameof(AuditConfig.OverrideTimeToBeReceived)} attribute in the {nameof(AuditConfig)} configuration section is discouraged and will be removed in the next major version. Switch to the code API by using `{nameof(EndpointConfiguration)}.AuditProcessedMessagesTo` instead.");
```

Comment:
Should we be more specific and mention the timeToBeReceived overload on that method?

https://github.com/Particular/NServiceBus/blob/18e4f48f34553206fc9ae9697ac12d93c19d6659/src/NServiceBus.Core/Audit/ConfigureAudit.cs#L16